src/pages/gateway/getting-started.md

```diff
@@ -30,14 +30,18 @@ You will also need to have:
 
 ## Configure your environment
 
-Once you have an Adobe IO account, you need to access the CLI tool to start configuring your APIs with the schema management service.
+Once you have an Adobe IO account, you can use the CLI interface to start configuring your APIs with the schema management service. 
```

Comment:
```suggestion
Once you have an Adobe IO account, you can use the command line interface (CLI) to start configuring your APIs with the schema management service. 
```

Anything but CLI interface, because that's repetitive.